packages/cashier/src/Components/Email/__tests__/email-sent.spec.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { fireEvent, screen, render } from '@testing-library/react';
+import EmailSent from '../email-sent';
+
+describe('<EmailSent />', () => {
+    const resendVerificationEmail = jest.fn();
+    const setIsResendClicked = jest.fn();
+
+    it('component should be rendered', () => {
+        const component = render(<EmailSent />);
```

Comment:
can we destructure this to get `container`?

Reply:
sure, thx for notice!